File: Fibonacci/fib.kt
fun main(args : Array<String>) {
  val n : Int = 40;
  println("fib(${n}) = ${fib(n)}");
}

fun fib(n: Int): Int{
  if(n==0||n==1) return 1;
  return fib(n-1)+fib(n-2);
}

File: Fibonacci/fib.m
#import <Foundation/Foundation.h>

int fib(int n){
    if(n == 0 || n == 1){
        return 1;
    }
    return fib(n - 1) + fib(n - 2);
}

int main(int argc, const char * argv[])
{

    @autoreleasepool {
        
        int n = 40;
        NSLog(@"fib(%d) = %d\n", n, fib(n));
    }
}
